tensorflow_addons/image/distance_transform_test.py

```diff
@@ -97,38 +97,37 @@ def test_image_with_invalid_dtype(self):
             # pylint: disable=bad-continuation
             with self.assertRaisesRegex(
                     TypeError, "`dtype` must be float16, float32 or float64"):
-                _ = distance_tranform_ops.euclidean_dist_transform(
+                _ = distance_transform_ops.euclidean_dist_transform(
                     image, dtype=output_dtype)
 
     def test_image_with_invalid_shape(self):
-        for invalid_shape in ([1], [2, 1], [2, 4, 4, 4, 1]):
```

Comment:
Why to remove the test case for 1D and 5D images?

Reply:
This [test](https://github.com/tensorflow/addons/blob/d16b4077de06ed0636a93fbe88adc4ed7faec096/tensorflow_addons/image/utils_test.py#L45) in `image/utils_test.py` takes care of that, since I'm calling `to_4D_image` in `euclidean_dist_tranform`.

Reply:
Oh, I see. We won't get any warning if the behavior of `to_4D_image` changes, however.

Reply:
As long as the behaviour of `to_4D_image` is to convert the image to a 4D Tensor, euclidean distance transform should function. The checks for other aspects like single channel, etc. are still in place. I can add back the test cases if you think it's a good idea.

Reply:
Both are fine to me. OK, let's remove it, and we can add the test case back later.